copilot/fastapi/tests/test_tvp.py

```
@@ -0,0 +1,141 @@
+        assert "is_reconciling" in data
+        assert "applications" in data
+        assert len(data["applications"]) == 1
+        assert data["applications"][0]["app_name"] == "test-app"
+        assert data["applications"][0]["namespace"] == "test-namespace"
+        assert data["applications"][0]["image"] == "test-image"
+        assert data["applications"][0]["tag"] == "v1.0.0"
+
+def test_trigger_reconciliation(test_client):
+    """Test the reconciliation trigger endpoint"""
+    # Mock the reconciliation function to avoid actual execution
+    with patch("app.tvp.reconcile_from_git") as mock_reconcile:
```

Ruff (0.8.2)

Local variable `mock_reconcile` is assigned to but never used

Remove assignment to unused variable `mock_reconcile`

(F841)